server/src/QRServer/db/migrations.py

```diff
@@ -23,6 +23,25 @@ def execute_migrations(c):
     if version <= 1:
         c.execute("alter table users add column comment varchar")
         _set_version(c, 2)
+    if version <= 2:
+        c.execute(
+            "create table matches ("
```

Comment:
Refactor this to hold winner/loser instead of p1/p2. This will make it easier to format recent battles returned to clients